src/func.py

```diff
@@ -0,0 +1,24 @@
+import exceptions
+
+
+class Math:
+    def __init__(self, num1, num2):
+        self.num1 = num1
+        self.num2 = num2
+
+    def FunSummation(self):
```

Comment:
Пожалуйста переименуй, функции должны называться с camel_case, а классы с PascalCase
```suggestion
    def fun_summation(self):
```